tests/backend/core/Security/Authenticator/BasicKeycloakUserDataTest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of the package demosplan.
+ *
+ * (c) 2010-present DEMOS E-Partizipation GmbH, for more information see the license file.
+ *
+ * All rights reserved
+ */
+
+namespace Tests\Core\Security\Authenticator;
+
+use demosplan\DemosPlanCoreBundle\ValueObject\BasicKeycloakUserData;
+use demosplan\DemosPlanCoreBundle\ValueObject\KeycloakUserDataInterface;
+use KnpU\OAuth2ClientBundle\Client\ClientRegistry;
+use League\OAuth2\Client\Token\AccessToken;
+use Tests\Base\FunctionalTestCase;
+
+class BasicKeycloakUserDataTest extends FunctionalTestCase
```

Comment:
I get an exception demosplan\DemosPlanCoreBundle\Exception\ValueObjectException : ValueObject is not locked! Please use $this->lock() first in BasicKeycloakUserDataTest:35 when I run this test

Reply:
e8842bada9095f45fa0d8828d66a94fc527d5456

Reply:
noo :-( I think they work and probably are correct. I do not see anywhere where the value object is locked